R/accounts.R

```diff
@@ -417,6 +417,17 @@ str.rsconnect_secret <- function(object, ...) {
   cat(" ", format(object), "\n", sep = "")
 }
 
+#' @export
+as.data.frame.rsconnect_secret <- function(x,
+                                           ...) {
```

Comment:
style nit/question: Is this indenting intentional? The deep indent for a line starting with `...` feels inconsistent with many of the other multi-line function signatures, which would place the `x` on a separate line.